services/frontend-v3/src/i18n/en_CA.json

```diff
@@ -385,12 +383,14 @@
   "sign.out": "Logout",
   "home": "Home",
 
-  "profile.addFriend": "Add Friend",
-  "profile.removeFriend": "Remove Friend",
-  "profile.removeFriend.confirm": "Are you sure you want to remove this user from your friend group ?",
+  "profile.addConnection": "Add connection",
+  "profile.removeConnection": "Remove connection",
```

Comment:
This info is already defined, aka the "search.results.cards.add.connection" and "search.results.cards.remove.connection" keys, could we somehow unite those keys